app/assets/javascripts/models/data/service_options.js.coffee

```diff
@@ -176,10 +176,14 @@ ns.ServiceOptions = do (ko, edsc = @edsc, KnockoutModel = @edsc.models.KnockoutM
       return true if @granuleAccessOptions().methods?.length == 0
       
       result = false
+      selected = false
```

Comment:
- `result = true if m.method()? && m.isValid()`
- `selected = true if m.method()?`

and the value we return `result && selected`

My comment on redundancy was the above, I don't think you need the `selected` value at all. It's check is just a subset of the check that was already there.